utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the utopianTree function below.
+def utopianTree(n): 
+
+    #my code starts here      
+    number_of_periods = n
+    print("number_of_periods arxh =",number_of_periods)
+    height = 1 
+    
+    for i in range(0,number_of_periods):       
```

Comment:
range always starts from zero, you don't need to define it

```python
for i in range(number_of_periods):
    pass
```

Is more clean

Reply:
Corrected